rapidproCSDAdapter.js

```diff
@@ -89,8 +90,9 @@ module.exports = function (config) {
           callback(null, converted, _orchestrations.concat(orchestrations))
         }
       }
-
-      rapidpro.getContacts(groupUUID, getContactsCallback([]))
+      
+		let url = rapidpro.contactsURL(groupUUID)
```

Comment:
`contactsURL` is not a public function so you can't use it like this. Is there a reason you need to do it like this rather than how it was before?